app/parse-siti-payment-file/build-payment-requests.js

```diff
@@ -3,9 +3,9 @@ const handleSitiDefects = require('./handle-siti-defects')
 const { buildInvoiceLines, isInvoiceLineValid } = require('./build-invoice-lines')
 const { v4: uuidv4 } = require('uuid')
 
-const buildPaymentRequests = (paymentRequests) => {
+const buildPaymentRequests = (paymentRequests, sourceSystem) => {
   return paymentRequests.map(paymentRequest => ({
-    sourceSystem: paymentRequest.sourceSystem,
+    sourceSystem,

```

Comment:
`sourceSystem` was mapped from the header line of each payment request for SFI Pilot.  That doesn't exist for Lump Sums so have changed so consistently takes the value from the Batch line.